integration-test/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "build": "netlify-build",
```

Comment:
where is this command coming from? it's not an installed package right? could you install it locally if it's something you have globally? how would linking with the current plugin code work?

Reply:
I think this was a local command I was running earlier when working with ts files, it ran using a local version of netlify build, I'll update since you'll probably want to run `netlify build` so it picks up the plugin locally

Reply:
ok, so we need to install `netlify-cli`?

Reply:
`netlify-build` is the binary of `@netlify/build`.

https://github.com/netlify/build/blob/58be3dff6f0d359f49264a8da66430752fdf7549/packages/build/package.json#L7

`netlify build` is a Netlify CLI command which calls `@netlify/build` and adds few Netlify CLI-specific things, such as retrieving the Site ID and API token set by `netlify init` and `netlify login`.

https://github.com/netlify/cli/blob/aef62e9e6e8a035936c692e5a1382b8890ae9f78/src/commands/build/index.js#L16

For testing purposes, I would personally recommend running `@netlify/build` directly (either programmatically or with the binary) without going through the Netlify CLI.